application/common/models/User.php

```diff
@@ -350,4 +376,56 @@ public function getVerifiedMethods()
     {
         return Method::findAll(['user_id' => $this->id, 'verified' => 1]);
     }
+
+    public function getPasswordMeta()
+    {
+        $thisUser = self::findOne(['id' => $this->id]);
```

Comment:
Is there a reason we can't just use `$this` here instead of having to retrieve a User object that has the id that this User object has?